src/entities/jsonld.ts

```diff
@@ -1,25 +1,66 @@
 import { EDC_CONTEXT } from "./context";
+import jsonld from "jsonld";
+
+export async function expand<T extends JsonLdObject>(body: any, newInstance: (() => T)): Promise<T> {
+  const expanded = await jsonld.expand(body);
+  var instance = Object.assign(newInstance(), expanded[0]);
+  instance._compacted = body;
+  return instance;
+};
+
+export async function expandArray<T extends JsonLdObject>(body: any, newInstance: (() => T)): Promise<T[]> {
+  const expanded = await jsonld.expand(body);
+  return (expanded as Array<any>).map((element, index) => {
+    var instance = Object.assign(newInstance(), element);
+    instance._compacted = body[index];
+    return instance;
+  });
+}
```

Comment:
I'd personally like to have these somewhere else than  the `entities/` folder. But I also believe there'll be some refactor after v0.2.0, so as long as we keep this in mind, I'm good to land this now without adding complications

Reply:
same here, but I really think we'll need some re-engineering, the idea would be to reach 0.2.0, bump assets to v3 and then do some housekeeping